extensions/org.eclipse.lsp4xml.extensions.emmet/src/main/java/org/eclipse/lsp4xml/extensions/emmet/emmetio/extract/ExtractAbbreviation.java

```diff
@@ -75,9 +75,9 @@ public static ExtractAbbreviationResult extractAbbreviation(String line, Integer
 				return null;
 			}
 			String abbreviation = (String) s.get("abbreviation");
-			Double location = (Double) s.get("location");
-			Double start = (Double) s.get("start");
-			Double end = (Double) s.get("end");
+			Number location = (Number) s.get("location");

```

Comment:
To support both JDK8 and JDK11, I suggets you to have a private  static method `getInt` which manages Number and Double like this:

```java
private static int getInt(Bindings bindings, String key) {
	Object result = bindings.get(key);		
	if (result instanceof Number) {
		return ((Number) result).intValue();
	}
	if (result instanceof Double) {
		return ((Double) result).intValue();
	}
	return 0;
}
```

After that you call it like this:

`int location = getInt(s, "location");`

Reply:
I don't believe this is necessary. `Double` and `Integer` (et al.) both implement `Number` (see https://docs.oracle.com/javase/8/docs/api/java/lang/Number.html), so the current change works on both platforms -- were this not the case the CI run would have failed.